include/glow/ExecutionEngine/ExecutionEngine.h

```diff
@@ -83,6 +83,8 @@ class ExecutionEngine final {
 
   /// Runs a single execution of the function.
   void run();
```

Comment:
Can you remove the old run method?

Reply:
I was planning to remove it once memory allocation is working in the new run/execute flow. This gives the community a chance to port this week while the allocation work is being done. If we remove it now it will break the API. If we go that route we should hold on the merge until after the email is sent out.